.github/ISSUE_TEMPLATE/false-positive---false-negative.md

```diff
@@ -29,7 +29,7 @@ Please provide a description of any known workarounds.
 
 ### Related information
 
-* SonarC# Version
+* C#/VB.NET SonarQube Plugins Version
```

Comment:
I wouldn't put `SonarQube` given it can be `SonarLint` or `SonarCloud` as well...